app/views/events/index.html.erb

```diff
@@ -33,7 +33,7 @@
               <p class="event-name"><%= link_to event.title, event %></a></p>
             </td>
             <td class="event-location">
-              <p class="event-venue"><a href="#"><%= event.venue.name %></a></p>
+              <p class="event-venue"><%= link_to(event.venue.name, venue_path(event.venue.id)) %></p>
```

Comment:
I think a path can take either an ID or an object, so we may be able to leave off the id and just use venue_path(event.venue)

Also, would it be good if we fatten up the models and make some methods so we don't have to chain a bunch of methods together? Not sure if it's worth it in this case but wouldn't be hard.